### PR TITLE
Link from README to the release script

### PR DESCRIPTION
**Pull request by ewjoachim**
Also mention that packages are released daily

### CODE REVIEW

File: README.md
Comment (Akuli) [inline README.md:36]: 0.900 has been released for a while now.

```suggestion
Also, starting from version 0.900, you can use mypy's `--install-types` option to automatically install
```

File: README.md
Comment (Akuli) [inline README.md:34]: If I understood correctly, the release always happens within 3 hours, but it would be good if another typeshed maintainer confirms this.

Comment (JelleZijlstra) [reply README.md:34]: I'm pretty sure 3 hours is right. Elsewhere you linked to the crontab and the cron runs every 3 hours.

Comment (ewjoachim) [reply README.md:34]: Oh you right. I read the cron too fast and saw every day at 3, where it's "every hours divisible by 3" so 8 times a day

Comment (ewjoachim) [reply README.md:34]: You can check how often the CI runs: https://github.com/typeshed-internal/stub_uploader/actions/workflows/update_stubs.yml

I was put off by the name of the workflow: "Daily update of stubs from typeshed to PyPI".

Comment (ewjoachim) [reply README.md]: Ok, fixed :)

**Comment by ewjoachim** [comment]
So logically from [this](https://github.com/typeshed-internal/stub_uploader/pull/40#issuecomment-982804898) comment, I would understand that this is not wanted. If that's so, I'm not in a position to do anything and we can close.

As a user of the said packages, though, I'm a little bit concerned by the idea the processes behind those packages would be "internal" and that making it hard to figure out where the release is being done is a feature and not a bug.

Of course, we could be in a situation where those packages are released manually and there would not be a way to audit how it's done. But given that there IS an open source script somewhere, I personally find some value in being able to find it on my own through documentation, and to study it.

With all that said, I'll let you decide how best you want to handle this :)

**Comment by JelleZijlstra** [comment]
I think it's fine to link from the typeshed README.

**Review by hauntsaninja — CHANGES_REQUESTED** [review]
`--install-types` doesn't actually look at PyPI and only works for a subset of typeshed packages. I recommend not mentioning it (e.g. see https://github.com/python/mypy/issues/10619 , https://github.com/python/mypy/blob/master/mypy/stubinfo.py). It also has some quirks (changes what versions of packages it installs over time, interactive by default, requires mypy cache and running mypy twice).

**Comment by ewjoachim** [comment]
I'd rather avoid back and forths on parts that are not my PR's subject :)

Can you @hauntsaninja and @Akuli give me a direction ? (asking because of https://github.com/python/typeshed/pull/6458#discussion_r759378210)

**Comment by JelleZijlstra** [comment]
I vote we just remove the part about `--install-types`. typeshed should aim to be type checker-agnostic anyway.

**Comment by ewjoachim** [comment]
Done :)

**Review by srittau — APPROVED** [review]

File: README.md
Comment (JelleZijlstra) [inline README.md:38]: Now this paragraph doesn't make sense, because it meant "similar to mypy". I would rewrite it as "Type checkers should be able to use these stub packages when installed. For more details, see the documentation for your type checker."

**Comment by ewjoachim** [comment]
Done :)

Maybe it might be worth it for someone who knows the project to make a full update of the documentation, fixing everything that needs to be fixed, but I suggest we do this on a different PR :)

**Comment by JelleZijlstra** [comment]
Yes, sorry for the back-and-forth here.

**Pull request merged**
